libraries/TGRSIAnalysis/TAngularCorrelation/TAngularCorrelation.cxx

```diff
@@ -418,7 +547,8 @@ TGraphAsymmErrors* TAngularCorrelation::CreateGraphFromHst(TH1* hst)
 
       // fill graph with point
       Int_t graphn = graph->GetN();
-      graph->SetPoint(graphn,TMath::Cos(angle),y);
+      if(!fold) graph->SetPoint(graphn,TMath::Cos(angle),y);
+      else if(fold) graph->SetPoint(graphn,angle,y);
```

Comment:
Are the elements in the folded angles array actually the cosine values already? Otherwise, I don't see a reason this needs to have if statements in it.